docs/designers-developers/developers/block-api/block-attributes.md

```diff
@@ -32,6 +32,20 @@ _Example_: Extract the `src` attribute from an image found in the block's markup
 }
 // { "url": "https://lorempixel.com/1200/800/" }
 ```
+#### `attribute` Type Validation
+
+Accepted values in the `type` field of an `attribute` MUST be one of the following:
+
+* null
+* boolean
+* object
+* array
+* number
+* string
+* integer
+	- Matches any number with a zero fractional part
+
+Additional details are found in the documentation for Wordpress's REST API [here](https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/).
```

Comment:
Here's a suggestion for improving the link:

```suggestion
See [Wordpress's REST API documentation](https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/) for additional details.
```

Reply:
Also, capital P :)

```suggestion
See [WordPress's REST API documentation](https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/) for additional details.
```

Reply:
hah, how'd I miss that? :-)